data/src/main/java/com/example/data/di/RepositoryModule.kt

```diff
@@ -0,0 +1,15 @@
+package com.example.data.di
+
+import com.example.data.repository.GithubRepositoryImpl
+import com.example.domain.repository.GithubRepository
+import dagger.Binds
+import dagger.Module
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+
+@InstallIn(SingletonComponent::class)
+@Module
+abstract class RepositoryModule {
+    @Binds
+    abstract fun bindGithubRepository(impl: GithubRepositoryImpl): GithubRepository
```

Comment:
이거 베이스에서 놓친 것 같은데
한번 확인 부탁합니다 민지🐑 
```suggestion
    @Binds
    @Singleton
    abstract fun bindGithubRepository(impl: GithubRepositoryImpl): GithubRepository
```